backend/app.ts

```diff
@@ -504,12 +521,12 @@ io.on(M.connectionMsg, function (socket: any): void {
 
   socket.on(
     M.getSolvedPuzzlesMsg,
-    async function ({ eloId }: { eloId: string }): Promise<void> {
-      logReceivedMessage(M.getSolvedPuzzlesMsg, { eloId });
-      const pseudoPlayer = await db.getPseudoPlayer(eloId);
-      if (pseudoPlayer) {
+    async function ({ idToken }: { idToken: string }): Promise<void> {
+      logReceivedMessage(M.getSolvedPuzzlesMsg, { idToken });
+      const Player = await db.getPlayer(idToken);
```

Comment:
I think `Player` should be lowercase, like `pseudoPlayer` was.

Reply:
done